### PR TITLE
[WIP] Add zh_CN localization for cutechess-gui.

### PR DESCRIPTION
**Pull request by Prcuvu**
Now GUI will be able to load translations by auto-detecting system locale, with a working zh_CN translation.

### CODE REVIEW

File: projects/gui/ui/timecontroldlg.ui [hunk 1]
```diff
@@ -170,7 +170,7 @@
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="m_incrementSpin">
        <property name="toolTip">
-        <string>Time increment per move in milliseconds</string>
+        <string>Time increment per move in seconds</string>
```
Comment (Prcuvu) [inline projects/gui/ui/timecontroldlg.ui]: This was proved a typo to correct.

**Comment by ilaripih** [comment]
That's a heroic effort, thanks! I'll test this later today.

**Comment by artoj** [comment]
Nice! Could the typos and adding of tr() functions be in a separate commit as they're clearly bugs.

**Comment by Prcuvu** [comment]
The GUI application doesn't do translation for strings in cutechess-lib. I know where I made it wrong, and I will do it all over again. Please don't merge this PR yet.

**Comment by Prcuvu** [comment]
Replaced by #344.

**Pull request closed without merging**